app/redux/reducers/SettingsReducer.js

```diff
@@ -3,17 +3,34 @@ import {
     PRESENT_END_SET_TIMER,
     DISMISS_END_SET_TIMER,
     LOGIN_SUCCESS,
-    UPDATE_SET_DATA_FROM_SERVER
+    UPDATE_SET_DATA_FROM_SERVER,
+    SET_DEFAULT_METRIC,
+    EDIT_DEFAULT_METRIC,
+    DISMISS_SET_METRIC
 } from 'app/ActionTypes';
 
 const defaultState = {
+    defaultMetric: 'kgs',
+    editingDefaultMetric: false,
```

Comment:
rename editingDefaultMetric to isEditingDefaultMetric

Reply:
Done